app/cmd/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/TudorHulban/GoLayouter/app/helpers/helpers"
+	"github.com/TudorHulban/GoLayouter/app/service"
+	"github.com/TudorHulban/GoLayouter/domain/objects"
+)
+
+func main() {
+	fileSource := os.Args[1]
+
+	content, err := helpers.ReadFile(fileSource)
+	if err != nil {
+		log.Print(err)
```

Comment:
do not use loq, normal fmt will do
how can we continue if there is an error on reading the file???

please use an os exit with 1

Reply:
addressed in d9c081de52e70f86c4ee0af1d15f3af69343fabf